MH4U Dex/CoreDataController.m

```diff
@@ -53,7 +54,7 @@ - (void)tryLoadSequence
     if ([[NSUserDefaults standardUserDefaults] boolForKey:NeedsReloadKey]) {
         NSLog(@"Resetting Core Data");
         [self resetCoreData];
-        NSManagedObjectContext *backgroundContext = [self backgroundManagedObjectContext];
+//        NSManagedObjectContext *backgroundContext = [self backgroundManagedObjectContext];
```

Comment:
Why don't you move all that into a method, and only add a call to the method as needed?